studio/components/prefixedSlugInput.module.css

```diff
@@ -0,0 +1,28 @@
+.wrapper {
+  display: flex;
+}
+
+.wrapper input {
+  padding-left: 0.5rem;
+  border-left: none;
+}
+
+.prefixWrapper + * * {
+  border-top-left-radius: 0 !important;
+  border-bottom-left-radius: 0 !important;
+}
+
+.prefixWrapper {
+  background-color: rgb(246, 246, 248);
+  border-radius: 4px 0 0 4px;
+  padding: 0.375rem 0.5rem 0.375rem 0.5rem;
+  border: 1px solid rgb(225, 226, 230);
+  border-right: none;
+}
+
+[data-scheme="dark"] .prefixWrapper {
+  background-color: rgb(25, 26, 36);
```

Comment:
hm.. could we add these colors in a global css file for studio? I believe Ane also introduced some colors in costum components?

Reply:
Can we do this in a separate PR?

Reply:
Yes, I think there are several places we want to update!